content/concepts/transports/webrtc.md

```diff
@@ -101,17 +101,85 @@ Authenticity is achieved by succeeding the
 
 <!-- TO ADD DIAGRAM -->
 
-### Coming soon: Browser-to-Browser
+### WebRTC W3C (Hole punching to achieve Browser-to-Browser connectivity)
 
-Eventually, libp2p will have support for communication between two
-browsers.
+Thanks to js-libp2p and rust-libp2p (complied to Wasm), libp2p can run in the browser environment.
```

Comment:
We don't use the wasm compilation for browser to browser, nor elsewhere, do we? Consider removing?
```suggestion
Thanks to js-libp2p and rust-libp2p (compiled to Wasm), libp2p can run in the browser environment.
```

Reply:
My intention with this line was to indicate it's possible to run a rust libp2p node in the browser (when compiled to Wasm), not to indicate browser to browser is supported in rust-libp2p. I can either make this more clear or delete this.

Reply:
It is not implemented yet. It is on the roadmap only.

https://github.com/libp2p/rust-libp2p/blob/master/ROADMAP.md#webrtc-support-browser-to-browser